### DISCUSSION TITLE
Add support for other wikis.

### DISCUSSION THREAD

**Discussion by tehguitarist** [xxyzz/WordDumb#220] [Ideas]
So when x-raying my collection, I noticed a few fandoms have migrated to their own dedicated wikis. Two that come to mind are the Coppermind wiki for Brandon Sandersons works (which are more comprehensive than the fandom pages) and the  Abidan Archives which tell you to visit their wiki directly on their fandom page. Given that these typically are still built off the wiki engine, would it be possible to add "at your own risk" support for these alternatives? If each integration is a custom one, rather than just passing a wiki scraper over them, then obviously this is higher work than reward, just thought I'd ask.

For reference here are a couple of the ones mentioned:
https://wiki.abidanarchive.com
https://coppermind.net/wiki/Coppermind:Welcome

**Comment by xxyzz** [comment]
You could try to put these wiki links in the plugin's settings window. If they use MediaWiki and also enable the APIs used here: https://github.com/xxyzz/WordDumb/blob/9e0fa4cc2beb32c9328dbce8fb491eacb50ea012/mediawiki.py#L223 then descriptions will be downloaded.

**Reply by tehguitarist** [reply]
Thanks, I’ll double check but I believe I was being blocked from entering in the url in the settings window because it didn’t conform to https://*.fandom.com.

I’ll try editing the config file as a workaround though.

**Reply by xxyzz** [reply]
I forget that fandom domain validator, will remove it later...

**Reply by xxyzz** [reply]
Other wikis could be added now, please download test plugin from https://github.com/xxyzz/WordDumb/actions/runs/9232815534

**Comment by tehguitarist** [comment]
Seems to work very well! Thank you!
